### PR TITLE
telemetry mappings

### PR DESCRIPTION
**Pull request by eristhea**
I chose the most pointless area to start tackling unmapped classes

this 70% mojmap, 30% my own understanding, so please be thorough

notes:
I really don't like `TelemetryPropertyMap`, which is imported from mojmap, but I couldn't come up with a better one
please make suggestions!

### CODE REVIEW

File: mappings/net/minecraft/client/util/telemetry/TelemetryProperty.mapping [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+		ARG 2 container
+	METHOD m_jadmvviu createUuid (Ljava/lang/String;Ljava/lang/String;)Lnet/minecraft/unmapped/C_vykgtyww;
+		ARG 0 id
+		ARG 1 exportKey
+	METHOD m_jvqgfioi createString (Ljava/lang/String;Ljava/lang/String;)Lnet/minecraft/unmapped/C_vykgtyww;
+		ARG 0 id
+		ARG 1 exportKey
+	METHOD m_maatndmi createBoolean (Ljava/lang/String;Ljava/lang/String;)Lnet/minecraft/unmapped/C_vykgtyww;
+		ARG 0 id
+		ARG 1 exportKyey
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/util/telemetry/TelemetryProperty.mapping]:
```suggestion
		ARG 1 exportKey
```

File: mappings/net/minecraft/client/util/telemetry/logging/TelemetryEventLogDirectory.mapping [hunk 2]
```diff
@@ -0,0 +1,68 @@
+CLASS net/minecraft/unmapped/C_yknlgtdk net/minecraft/client/util/telemetry/logging/TelemetryEventLogDirectory
+	FIELD f_ifeiikjk root Ljava/nio/file/Path;
+	FIELD f_ohxcsiib COMPRESS_BUFFER_SIZE I
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/util/telemetry/logging/TelemetryEventLogDirectory.mapping]:
```suggestion
	FIELD f_ohxcsiib COMPRESSED_BUFFER_SIZE I
```
?

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
 	FIELD f_bvwetahy TELEMETRY_SENDER_EXECUTOR Ljava/util/concurrent/Executor;
+	FIELD f_duxfibma propertyMap Lnet/minecraft/unmapped/C_tupaamrk;
 	FIELD f_dzalcfzr telemetrySenderThreads Ljava/util/concurrent/atomic/AtomicInteger;
+	FIELD f_jgobxttp logManager Ljava/util/concurrent/CompletableFuture;
+	FIELD f_qsuuyhhu userApi Lcom/mojang/authlib/minecraft/UserApiService;
+	FIELD f_xkinpwcy logDirectory Ljava/nio/file/Path;
+	METHOD <init> (Lnet/minecraft/unmapped/C_ayfeobid;Lcom/mojang/authlib/minecraft/UserApiService;Lnet/minecraft/unmapped/C_cwfizcnb;)V
+		ARG 2 userApi
+		ARG 3 session
+	METHOD m_botbklbs logDirectory ()Ljava/nio/file/Path;
```
Comment (IotaBread) [inline mappings/net/minecraft/client/util/telemetry/TelemetryManager.mapping]:
```suggestion
	METHOD m_botbklbs getLogDirectory ()Ljava/nio/file/Path;
```

File: mappings/net/minecraft/client/util/telemetry/TelemetryProperty.mapping [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+		ARG 2 codec
+		ARG 3 exporter
+	METHOD m_usjtckev text ()Lnet/minecraft/unmapped/C_npqneive;
+	METHOD m_ypyjfwfa id ()Ljava/lang/String;
+	CLASS C_afeghidc ServerType
+		FIELD f_hfcrjewu key Ljava/lang/String;
+		METHOD <init> (Ljava/lang/String;ILjava/lang/String;)V
+			ARG 3 key
+	CLASS C_bqxexqea Exporter
+		METHOD apply export (Lcom/mojang/authlib/minecraft/TelemetryPropertyContainer;Ljava/lang/String;Ljava/lang/Object;)V
```
Comment (IotaBread) [inline mappings/net/minecraft/client/util/telemetry/TelemetryProperty.mapping]:
```suggestion
		METHOD apply (Lcom/mojang/authlib/minecraft/TelemetryPropertyContainer;Ljava/lang/String;Ljava/lang/Object;)V
```

File: mappings/net/minecraft/client/util/telemetry/TelemetryPropertyMap.mapping [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+	METHOD <init> (Ljava/util/Map;)V
+		ARG 1 propertyMap
+	METHOD m_abokrznm keySet ()Ljava/util/Set;
+	METHOD m_feetwqfc createCodec (Ljava/util/List;)Lcom/mojang/serialization/Codec;
+		ARG 0 propertyMap
+	METHOD m_fkeecsdt getValue (Lnet/minecraft/unmapped/C_vykgtyww;)Ljava/lang/Object;
+		ARG 1 property
+	METHOD m_ihalzlli builder ()Lnet/minecraft/unmapped/C_tupaamrk$C_ivdfigoc;
+	CLASS C_citwzovz
+		METHOD decode decode (Lcom/mojang/serialization/DynamicOps;Lcom/mojang/serialization/MapLike;)Lcom/mojang/serialization/DataResult;
```
Comment (IotaBread) [inline mappings/net/minecraft/client/util/telemetry/TelemetryPropertyMap.mapping]:
```suggestion
		METHOD decode (Lcom/mojang/serialization/DynamicOps;Lcom/mojang/serialization/MapLike;)Lcom/mojang/serialization/DataResult;
```

File: mappings/net/minecraft/client/util/telemetry/TelemetryPropertyMap.mapping [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+			ARG 2 data
+		METHOD encode (Ljava/lang/Object;Lcom/mojang/serialization/DynamicOps;Lcom/mojang/serialization/RecordBuilder;)Lcom/mojang/serialization/RecordBuilder;
+			ARG 1 propertyMap
+			ARG 3 builder
+		METHOD keys keys (Lcom/mojang/serialization/DynamicOps;)Ljava/util/stream/Stream;
```
Comment (IotaBread) [inline mappings/net/minecraft/client/util/telemetry/TelemetryPropertyMap.mapping]:
```suggestion
		METHOD keys (Lcom/mojang/serialization/DynamicOps;)Ljava/util/stream/Stream;
```

File: mappings/net/minecraft/client/util/telemetry/event/TelemetryEventSender.mapping [hunk 7]
```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/unmapped/C_wzhvxvch net/minecraft/client/util/telemetry/event/TelemetryEventSender
+	FIELD f_qkyauukq UNIMPLEMENTED_SENDER Lnet/minecraft/unmapped/C_wzhvxvch;
+	METHOD decorate create (Ljava/util/function/Consumer;)Lnet/minecraft/unmapped/C_wzhvxvch;
```
Comment (IotaBread) [inline mappings/net/minecraft/client/util/telemetry/event/TelemetryEventSender.mapping]:
```suggestion
	METHOD decorate (Ljava/util/function/Consumer;)Lnet/minecraft/unmapped/C_wzhvxvch;
```

File: mappings/net/minecraft/client/util/telemetry/event/TelemetryEventSender.mapping [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+		ARG 1 consumer
+	METHOD m_fizersuf (Lnet/minecraft/unmapped/C_tmmyxvlq;Ljava/util/function/Consumer;)V
+		ARG 0 type
+		ARG 1 consumer
+	METHOD m_pdonlvni (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_tmmyxvlq;Ljava/util/function/Consumer;)V
+		ARG 2 type
+		ARG 3 secondConsumer
+	METHOD send send (Lnet/minecraft/unmapped/C_tmmyxvlq;Ljava/util/function/Consumer;)V
```
Comment (IotaBread) [inline mappings/net/minecraft/client/util/telemetry/event/TelemetryEventSender.mapping]:
```suggestion
	METHOD send (Lnet/minecraft/unmapped/C_tmmyxvlq;Ljava/util/function/Consumer;)V
```

**Review by IotaBread — APPROVED** [review]

File: mappings/net/minecraft/client/util/telemetry/TelemetryProperty.mapping (hunk 4 above)
Comment (eristhea) [reply mappings/net/minecraft/client/util/telemetry/TelemetryProperty.mapping]: oh no, the only deobfuscated rename I could remember was `log()`

**Review by supersaiyansubtlety — APPROVED** [review]

**Pull request merged**